cmd/influxd/server_integration_test.go

```diff
@@ -868,6 +903,56 @@ func TestClientLibrary(t *testing.T) {
 	}
 }
 
+func Test_ServerSingleGraphiteIntegration(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	nNodes := 1
+	basePort := 8390
+	testName := "graphite integration"
+	dir := tempfile()
+	now := time.Now().UTC().Round(time.Millisecond)
+	c := main.NewConfig()
+	g := main.Graphite{
+		Enabled:  true,
+		Database: "graphite",
+		Protocol: "TCP",
+	}
+	c.Graphites = append(c.Graphites, g)
+
+	t.Logf("Graphite Connection String: %s\n", g.ConnectionString(c.BindAddress))
+	nodes := createCombinedNodeCluster(t, testName, dir, nNodes, basePort, c)
+
+	createDatabase(t, testName, nodes, "graphite")
+	createRetentionPolicy(t, testName, nodes, "graphite", "raw")
```

Comment:
I guess it's in the implementation -- and can be changed by the config params. If so, change it in the config params so that config value is tested as being effective.